package.json

```diff
@@ -19,7 +19,7 @@
   "license": "ISC",
   "dependencies": {
     "cids": "^0.8.3",
```

Comment:
You need to update this to the latest (1.1.5) as it also has dependencies to multicodec.

Reply:
Once updated, you also have to change CID to use `bytes` instead of `buffer` (looks like they changed interface)

<img width="608" alt="Screenshot 2021-02-04 at 17 48 42" src="https://user-images.githubusercontent.com/2630/106933801-42567780-6711-11eb-953f-ba506aebd0db.png">